pkgs/development/tools/misc/opengrok/default.nix

```diff
@@ -1,10 +1,10 @@
-{ fetchurl, stdenv, jre, ctags, makeWrapper, coreutils, git }:
+{ stdenv, fetchurl, jre, ctags, makeWrapper, coreutils, git }:
 
 stdenv.mkDerivation rec {
   name = "opengrok-${version}";
   version = "1.0";
 
-  # 1.0 is the latest distributed as a .tar.gz file.
+  # 0.12.1.5 is the latest distributed as a .tar.gz file.
   # Newer are distribued as .zip so a source build is required.
 
```

Comment:
Now that you've changed the `src` attribute to a more standard looking fetch, you can probably delete this whole comment and the one following it.

Reply:
Done